### PR TITLE
Br 10

### PR DESCRIPTION
**Pull request by MaliukDaria**

### CODE REVIEW

File: src/main/java/com/dev/cinema/annotation/CustomEmailValidator.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+public class CustomEmailValidator implements ConstraintValidator<CustomEmailConstraint, String> {
+    @Override
+    public void initialize(CustomEmailConstraint constraintAnnotation) {
+
+    }
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        return email != null && email.matches("^(.+)@(.+)$");
```
Comment (andromash) [inline src/main/java/com/dev/cinema/annotation/CustomEmailValidator.java]: REGEX pattern has to be in constant checklist says

File: src/main/java/com/dev/cinema/annotation/FieldsMatchValidator.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    public boolean isValid(Object value, ConstraintValidatorContext context) {
+        Object fieldValue = new BeanWrapperImpl(value)
+                .getPropertyValue(field);
+        Object fieldMatchValue = new BeanWrapperImpl(value)
+                .getPropertyValue(fieldMatch);
+        if (fieldValue != null) {
+            return fieldValue.equals(fieldMatchValue);
+        } else {
+            return fieldMatchValue == null;
+        }
```
Comment (andromash) [inline src/main/java/com/dev/cinema/annotation/FieldsMatchValidator.java]: you can simplify this if-else construction with ternary operator

**Review by andromash — COMMENTED** [review]
Good job!

File: src/main/java/com/dev/cinema/annotation/CustomEmailValidator.java (hunk 1 above)
Comment (MaliukDaria) [reply src/main/java/com/dev/cinema/annotation/CustomEmailValidator.java]: done

**Review by andromash — APPROVED** [review]

**Review by dmytro-hryhoruk — APPROVED** [review]
Good job!

**Pull request merged**